packages/@aws-cdk/aws-pipes-targets-alpha/lib/api-destination.ts

```diff
@@ -0,0 +1,80 @@
+import { IInputTransformation, IPipe, ITarget, TargetConfig } from '@aws-cdk/aws-pipes-alpha';
+import { IApiDestination } from 'aws-cdk-lib/aws-events';
+import { IRole, PolicyStatement } from 'aws-cdk-lib/aws-iam';
+
+/**
+ * EventBridge API destination target properties.
+ */
+export interface ApiDestinationTargetParameters {
+  /**
+   * The input transformation to apply to the message before sending it to the target.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargetparameters.html#cfn-pipes-pipe-pipetargetparameters-inputtemplate
+   * @default - none
+   */
+  readonly inputTransformation?: IInputTransformation;
+
+  /**
+   * The headers to send as part of the request invoking the EventBridge API destination.
+   *
+   * The headers are merged with the headers from the API destination.
+   * If there are conflicts, the headers from the API destination take precedence.
```

Comment:
@nmussy I added this note about precedence. From the logs we can see that x-api-key is abc123, not apiKeyFromHeaderParams.

Reply:
I'm not sure whether this is enough to indicate to users which value will take precedence. Is there somewhere we can print a warning when there are two identical headers provided, which explicitly lets the user know that the value is being set twice, and which of those values it's using?

Reply:
@redwheeler3 jfyi. I think it's worth mentioning in Docs and/or Console about precedence regarding conflict headers.